play-framework/websockets/app/actors/Messenger.java

```diff
@@ -0,0 +1,114 @@
+package actors;
+
+import akka.actor.AbstractActor;
+import akka.actor.ActorRef;
+import akka.actor.PoisonPill;
+import akka.actor.Props;
+import akka.event.Logging;
+import akka.event.LoggingAdapter;
+import akka.http.javadsl.Http;
+import akka.http.javadsl.marshallers.jackson.Jackson;
+import akka.http.javadsl.model.HttpMessage;
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.model.HttpResponse;
+import akka.stream.Materializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import dto.MessageDTO;
+import dto.RequestDTO;
+import utils.MessageConverter;
+
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.UUID;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Created by alfred on 02 Mar 2019
+ */
```

Comment:
Let's remove the author comments here, please.

Reply:
Okay sure. I have removed these everywhere they occurred.